wails-frontend/src/utils/massaFormat.tsx

```diff
@@ -37,16 +43,22 @@ export function toNanoMASS(str: string): number {
  */
 
 export function formatStandard(
-  num: number,
+  num: string | number,
   unit = Unit.MAS,
   maximumFractionDigits = 2,
 ): string {
-  const numInMas = unit === Unit.MAS ? num : Number(toMAS(num));
+  const numInMas = unit === Unit.MAS ? num : toMAS(num);
+  // Create an options object for toLocaleString
   const locale = localStorage.getItem('locale') || 'en-US';
-  return numInMas.toLocaleString(locale, {
+  const options = {
     minimumFractionDigits: 2,
     maximumFractionDigits,
-  });
+  };
+
+  // Format the number as a string with separators
+  const formattedNum = numInMas.toLocaleString(locale, options);
+
+  return formattedNum;
```

Comment:
why modify this code to do the same thing ? you can keep the comment but the previous version was more clean imho...